include/LMS1xx/lms_structs.h

```diff
@@ -203,4 +203,24 @@ struct scanData
   uint16_t rssi2[1082];
 };
 
+// TODO consider to use those for LMS as well
```

Comment:
Jup, but as described in the PR description, this will require a full rewrite of the LMS1xx code as well, which we will not tackle until we need to write a proper driver for a non-prototype version with a definitely fixed protocol ;-)